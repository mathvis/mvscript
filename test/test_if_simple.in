if (a > 0) b;
