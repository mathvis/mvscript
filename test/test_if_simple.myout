Decl (IfBlock (Operation (GreaterThan (VarIdentifier "a") (Type (Int 0)))) (Expr (VarIdentifier "b")) Nothing)
