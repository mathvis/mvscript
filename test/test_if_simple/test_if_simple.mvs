let a = 0;
let b = 0;
if (a > 0) b;
