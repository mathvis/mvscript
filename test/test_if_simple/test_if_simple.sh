#!/bin/bash
helix test_if_simple.mvs test_if_simple.out --vsplit
