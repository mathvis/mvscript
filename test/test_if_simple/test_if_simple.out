Decl (Variable (VarIdentifier "a") (Just int) (Just (Type (Int 0))))
Decl (Variable (VarIdentifier "b") (Just int) (Just (Type (Int 0))))
Decl (IfBlock (Operation (GreaterThan (VarIdentifier "a") (Type (Int 0)))) (Expr (VarIdentifier "b")) Nothing)
